clients/python-wrapper/tests/integration/test_sanity.py

```diff
@@ -31,6 +32,23 @@ def test_repository_sanity(storage_namespace, setup_repo):
         repo.delete()
 
 
+def test_branch_sanity(storage_namespace, setup_repo):
+    _, repo = setup_repo
+    branch_name = "test_branch"
+
+    main_branch = repo.branch("main")
+    new_branch = repo.branch(branch_name).create("main")
+    assert new_branch.repo_id == repo.properties.id
+    assert new_branch.id == branch_name
+    assert new_branch.head().id == main_branch.head().id
+    # TODO(Guys): add steps for creating an object, committing, heading and reverting
```

Comment:
Can you open an issue for this? :)

Reply:
https://github.com/treeverse/lakeFS/issues/7022